app/scripts/lib/multichain-api/provider-authorize/handler.js

```diff
@@ -64,7 +66,14 @@ export async function providerAuthorizeHandler(req, res, _next, end, hooks) {
     );
   }
 
+  const networkClientIdsAdded = [];
+
```

Comment:
1. What are networks requested
2. Find which ones are missing in the wallet currently
3. Are there scopedProperties to fallback to
4. If there are, validate them
5. If they are valid we send them as `networksToPropose` on the approval controller request
6. If some are invalid we don't include them in that `networksToPropose`, but if we have a popular network configuration we can send that instead